source/process/release-process.md

```diff
@@ -184,10 +184,13 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - Contact owners of [community installers](http://www.mattermost.org/installation/) or submit [PRs](https://github.com/tommyvn/mattermost-heroku/commit/94f7c5c0c5d7d2672fb6d62b6a560b4b5c1b5131) to update install version number
     - Close GitHub meta ticket for the release
     - Finalize docs
+      - If reviews are not complete, hold a 30 minute doc review meeting with PMs
       - Merge the docs release branch to master and verify all changes on docs.mattermost.com once the build is up
       - Submit a correction PR for any incorrect formatting or other errors missed during the intitial review
     - Update [MVP page](https://www.mattermost.org/mvp/) with the most valuable contributor of the release
-    - Submit GitLab MR for `config/software/mattermost.rb` to take next Mattermost version in the Omnibus. The MR should include changes to the `default_version` and `md5` sum (see [example](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/933/diffs)
+    - Submit GitLab MR to take next Mattermost version in the Omnibus (see [example](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/933/diffs)):
```

Comment:
Propose using 3.4 MR as example: https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/998

If you read the feedback in the 3.3 thread it had unnecessary info and info left out about necessary changes to gitlab.rb, so 3.4 is the better example.